packages/code-du-travail-frontend/src/modules/contributions/queries.ts

```diff
@@ -42,27 +38,74 @@ export const fetchContributions = async <
     .filter((source) => source !== undefined);
 };
 
+const formatContribution = (
+  contribution: ContributionElasticDocument | undefined
+): Contribution | undefined => {
+  if (!contribution) {
+    return undefined;
+  }
+  return {
+    ...contribution,
+    isGeneric: contribution.idcc === "0000",
+    isNoCDT: contribution?.type === "generic-no-cdt",
+    relatedItems: [
+      {
+        title: "Articles liés",
+        items: contribution.linkedContent.map((linked) => {
+          if (!isSource(linked.source)) {
+            throw new Error("la source est incorrecte");
```

Comment:
```suggestion
            Sentry.sendMessage("la source de ${JSON.stringify(linked)} est incorrecte");
```

Ne pas throw mais envoyé un message sentry et ne pas le traiter.